.github/CODEOWNERS

```diff
@@ -70,11 +70,14 @@
 /addons/mrp_subcontracting_account/ @odoo/rd-logistics
 /addons/mrp_subcontracting_dropshipping/ @odoo/rd-logistics
 
+/addons/payment*/ @odoo/rd-payment
```

Comment:
How many modules there are?
Due to precedence rules with this feature it's better to keep things explicit and avoid using wildcard in module names if possible.
(note: otherwise by guideline wildcard rules in modules have to be at the top of the file)

Reply:
14 modules, I can list them if you prefer, but since it's a clear rule for the whole scope, I found it cleaner like this.
But I'll move it at the top of the file